ddtrace/ddtrace.go

```diff
@@ -72,13 +72,23 @@ type Span interface {
 	// a representative name for a group of spans (e.g. "grpc.server" or "http.request").
 	SetOperationName(operationName string)
 
-	// BaggageItem returns the baggage item held by the given key.
-	BaggageItem(key string) string
```

Comment:
Is this a breaking change to an existing API?

Reply:
yes, I think so... I think maybe I should revert this change. But I also think it's slightly counterintuitive to call getBaggage `baggageItem`. Regardless, I think sticking to the original is better. And, in the future, we could possibly deprecate this name and change it to `getBaggageItem` depending on what people think

Reply:
In terms of breaking changes, yeah reverting this for now is probably a good idea.

But regarding language convention, I would ask the Go guild for guidance. I don't know if this is common in Go. Just to give you a different example, in .NET we are exposing baggage as an `IDictionary<string, string>`, so the API doesn't use functions and there is no `get` or `set` in the name, but it's functionally equivalent:
```csharp
// set
Baggage.Current["name"] = "value";

// get
var value = Baggage.Current["name"];
```

(that's unrelated to the fact that our baggage is separate from spans)

Reply:
As Lucas said, this change would break backwards compatibility. A few notes: 
1. The existing BaggageItem and SetBaggageItem methods already handle the single baggage item operations described in the RFC. You can make small changes within these methods in the span.go file if necessary. However, if the changes will break backward compatibility, we can instead introduce new methods to the interface to meet the RFC requirements.
2. In Go, it's conventional to avoid using "Get" in function names for getters. So, instead of naming the function GetBaggageItem, we would simply call it BaggageItem.

Reply:
The existing `BaggageItem` and `SetBaggageItem` have not been changed. I've only added `GetAllBaggageItems`, `RemoveBaggage` and `RemoveAllBaggageItems`. There's no need to edit or add to the existing functions as they function according to the RFC. I'm not sure about the naming for the new functions. Those can be discussed in the guild meeting!